docs/guides/developer/docs/architecture/api/playlists-api.md

```diff
@@ -0,0 +1,215 @@
+[TOC]
+
+# Information
+
+The Playlists API is available since API version 1.11.0.
+
+### GET /api/playlists/playlist.json
+
+Returns a playlist.
+
+__Response__
+
+`200 (OK)`: A playlist as JSON.
+
+| Field              | Type                        | Description                                               |
+|--------------------|-----------------------------|-----------------------------------------------------------|
+| `id`               | [`long`](types.md#basic)    | The playlist identifier                                   |
+
+
+__Example__
+
+```json
+{
+  "playlist": {
+    "id": 551,
+    "organization": "mh_default_org",
+    "entries": [
+      {
+        "id": 553,
+        "eventId": "ID-about-opencast",
+        "type": "EVENT"
+      },
+      {
+        "id": 554,
+        "eventId": "ID-3d-print",
+        "type": "EVENT"
+      }
+    ],
+    "title": "Opencast Playlist",
+    "description": "This is a playlist about Opencast",
+    "creator": "Opencast",
+    "updated": 1701854481056,
+    "accessControlEntries": {
+      "allow": true,
+      "role": "ROLE_USER_BOB",
+      "action": "read"
+    }
+  }
+}
+```
+
+### GET /api/playlists/playlists.json
+
+Get playlists. Playlists that you do not have read access to will not show up.
+
+__Response__
+
+`200 (OK)`: A JSON object containing an array.
+
+| Field                    | Type                          | Description                                                  |
+|--------------------------|-------------------------------|--------------------------------------------------------------|
+| `limit`                  | [`int`](types.md#basic)       | The maximum number of results to return for a single request |
+| `offset`                 | [`int`](types.md#basic)       | The index of the first result to return                      |
+| `sortByUpdated`          | [`boolean`](types.md#basic)   | Sort the results based on updated field                      |
+| `sortByUpdatedAscending` | [`boolean`](types.md#basic)   | If sorting by updated, should it be ascending?               |
```

Comment:
Feels more natural to have one `sort` argument that can be non-existent or "updated-ac" or "updated-desc". Yes? No?

Reply:
I have no opinion on this.

Reply:
Or the way it's implemented in the other endpoints. https://docs.opencast.org/r/15.x/developer/#architecture/api/usage/#sorting
`GET /api/playlists?sort=updated:ASC,<XYZ>:DESC`

Reply:
Following our own documentation seems like a pretty good choice, thanks :D